src/pages/JoinModal/JoinModal.scss

```
@@ -0,0 +1,126 @@
+@import '../../styles/variables.scss';
+
+.zarnaraCreateContainer {
+  background-color: white;
+  width: 650px;
+  height: 450px;
+  display: flex;
+  justify-content: center;
+  .zaranaraCreateAccount {
+    background-color: white;
+    width: 350px;
+    height: 450px;
+    .createAccountContainer {
+      padding-top: 10px;
+      display: flex;
+      justify-content: center;
+      margin-bottom: 5px;
+      span {
+        height: 30px;
+        font-size: 12px;
+        font-weight: bold;
+      }
+    }
+    .choiceKindOfAccount {
+      width: 340px;
+      display: flex;
+      justify-content: center;
+      .common {
+        span {
+          font-size: 15px;
+        }
+      }
+      .company {
+        span {
+          font-size: 15px;
+        }
+      }
+    }
```

새로운 선택자 간에는 한 줄 띄워서 가독성을 높여주세요!
```suggestion
    }
    
    .choiceKindOfAccount {
      width: 340px;
      display: flex;
      justify-content: center;
      
      .common {
        span {
          font-size: 15px;
        }
      }
      
      .company {
        span {
          font-size: 15px;
        }
      }
    }
```

---

그리고 동일한 style을 가지는 span 태그가 있는데, 해당 태그에 동일한 className을 부여하는 방식으로 하면 될 것 같네요!